README.md

```diff
@@ -1,3 +1,6 @@
+#Trello Link
```

Comment:
@danitacheny great start - thank you for setting up your Trello board, getting me the link and adding it to your PR.

One note about Markdown syntax - to use the octothorpe `#` as a header indicator, add a space between it and the following text, e.g.

```md
# Trello Link
```

will render as:

# Trello Link

Reply:
https://help.github.com/articles/basic-writing-and-formatting-syntax/